xhprof_lib/utils/xhprof_runs.php

```diff
@@ -442,11 +442,11 @@ public function save_run($xhprof_data, $type, $run_id = null, $xhprof_details =
 		}
 			
         
-	$url   = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : $_SERVER['PHP_SELF'];
+	$url   = isset($_SERVER['REQUEST_URI']) ? $_SERVER["REQUEST_URI"] : implode(' ', $_SERVER['argv']);
```

Comment:
Please change to:

```php
$url   = PHP_SAPI === 'cli' ? implode(' ', $_SERVER['argv']) : $_SERVER['REQUEST_URI'];
```

---

Compared to your original change I've replaced `isset($_SERVER['REQUEST_URI'])` with `PHP_SAPI === 'cli'` to clearly indicate, that request url isn't set in CLI.

Reply:
Ok, done